pycsw/wsgi.py

```diff
@@ -205,6 +207,12 @@ def get_configuration_path(process_environment, request_environment,
         )
     if not os.path.isabs(configuration_path):
         configuration_path = os.path.join(pycsw_root, configuration_path)
+
+    # brute force try default.cfg in the installation directory
+    default_cfg = os.path.join(pycsw_root, 'default.cfg')
+    if os.path.isfile(default_cfg):
```

Comment:
Sorry @tomkralidis maybe I am not making myself clear.

What I mean is:

- Up to line 210 we are defining the `configuration_path` variable by probing for several probable locations (`config` kvp param, `PYCSW_CONFIG` in the environment)
- Then on line 213 we check for the presence of `default_cfg`, which might hold another path different than the one found for `configuration_path`
- If `default_cfg` does exist we disregard whatever value had been previously probed from the request or the environment and just use this default. This renders the other methods of specifying the configuration pointless, especially because the `default_cfg` file is very likely to indeed exist.

In my opinion we should be testing for the existence of the path that was found by probing the request and the environment (`configuration_path`) first and use that if it exists. If this path does not exist then we'd gather the default value and use it.

Sorry for being such a PITA ;)